control-plane/agents/src/bin/core/controller/policies/rebuild_policies.rs

```diff
@@ -16,11 +16,12 @@ use std::time::Duration;
 */
 
 // time constants, in seconds
-const TWAIT_SPERF: std::time::Duration = Duration::new(600, 0);
-const TWAIT_SAVAIL: std::time::Duration = Duration::new(300, 0);
-const TWAIT_ZERO: std::time::Duration = Duration::new(0, 0);
+pub const TWAIT_SPERF: std::time::Duration = Duration::new(600, 0);
```

Comment:
please fix doc comments above

Reply:
nit: this file can simply be called rebuild.rs since we're already within policies.